WasatchNET/EEPROM.cs

```diff
@@ -1334,22 +1376,12 @@ public bool read()
                         maxLaserPowerMW = ParseData.toFloat(pages[3], 28);
                         minLaserPowerMW = ParseData.toFloat(pages[3], 32);
                         laserExcitationWavelengthNMFloat = ParseData.toFloat(pages[3], 36);
-
                         if (format >= 5)
                         {
                             minIntegrationTimeMS = ParseData.toUInt32(pages[3], 40);
                             maxIntegrationTimeMS = ParseData.toUInt32(pages[3], 44);
                         }
 
-                        if (format >= 7)

```

Comment:
note line 1214 still has "if (format > FORMAT)", which I think we said should be "if (format == 0xff)".

Reply:
The STA EEPROM in general is still sort of wonky, but I do think this is how we'll want it ultimately